app/scripts/modules/core/src/pipeline/executions/execution/ExecutionMarker.tsx

```diff
@@ -43,17 +39,7 @@ export class ExecutionMarker extends React.Component<IExecutionMarkerProps, IExe
     };
   }
 
-  private hydrate = (): void => {
```

Comment:
moving this hydration down to the label itself (which already had some redundant hydration code)